Rakefile

```diff
@@ -824,6 +827,38 @@ end
 installed_bzr = bzr_tasks.installed_path
 
 # ------------------------------------------------------------------------------
+# Git Credentials Helper
+# ------------------------------------------------------------------------------
+
+class DownloadOnlyTasks < BundleDependencyTasks
+
+  # Make the installed file executable?
+  attr_accessor :is_executable
+
+  def define_tasks
+    define_download_task
+    download_task
+    install_task
+  end
```

Comment:
This actually performs the download and installation tasks, whereas it should only define tasks that can do that.